lms/views/basic_lti_launch.py

```diff
@@ -362,6 +362,6 @@ def set_canvas_focused_user(self):
             display_name = "(Couldn't fetch student name)"
 
         # TODO! - Could/should this be replaced with a GradingInfo lookup?
-        self.context.hypothesis_config.update(
+        self.context.js_config["hypothesisClient"].update(
```

Comment:
Reaching inside this data structure feels like a signal this should be wrapped up in the service when that comes along.

Reply:
Yep. I think as we discussed, the aim will be for nothing to be mutating the config directly other than the service itself. And then views or whatever will call the service to have the config mutated for them